pySPT/preAnalysis/precision_noGUI.py

```diff
@@ -0,0 +1,429 @@
+"""
+@author: Johanna Rahm
```

Comment:
adjust docstring description (what is the difference to precision.py and why was it created & add your name under authors)